src/main/python/pre_fx_data.py

```diff
@@ -206,6 +210,47 @@ def load_data_set_from_nwb(self, path: str):
                 err
             )
 
+    def extract_manual_sweep_states(self):
+        """ Extract manual sweep states in the format schemas.ManualSweepStates
+        from PreFxData
+        """
+
+        return [
+            {
+                "sweep_number": 25,
+                "sweep_state": "failed"
+            },
+            {
+                "sweep_number": 26,
+                "sweep_state": "failed"
+            },
+
+               ]
+
+    def save_manual_states_to_json(self, path: str):
+
+        json_data = {
+            "input_nwb_file": self.nwb_path,
+            "stimulus_ontology_file": self.ontology_file,
+            "manual_sweep_states": self.extract_manual_sweep_states(),
+            "qc_criteria": self._qc_criteria,
+            "ipfx_version": ipfx.__version__
+        }
+
+        try:
```

Comment:
what about the case where the selected path cannot be opened for writing?

Reply:
I added an exception handling.

Reply:
btw you can write multiple excepts in a block:
```
try:
    raise ValueError()
except TypeError:
    print("type?")
except ValueError:
    print("value?")
```